pkg/mixin/install.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	DefaultFeedUrl = "https://cdn.deislabs.io/porter/atom.xml"
+	DefaultFeedUrl = "https://cdn.porter.sh/atom.xml"
```

Comment:
This should be /mixins/atom.xml right? Or are we not ready to have that be our default feed URL yet for installing mixins?

Reply:
You're right; updated.